vignettes/social_smell_showcase.Rmd

```diff
@@ -94,7 +97,16 @@ As stated in the introduction, we need both git log and at least one communicati
 
 ## Parse Gitlog
 
+Recommended perceval version: 0.12.24
 To get started, we use the `parse_gitlog` function to extract a table from the git log. You can inspect the `project_git` variable to inspect what information is available from the git log. 
+Reminder: the parse_gitlog function has a path.expand function, Expand paths (e.g. "~/Desktop" => "/Users/someuser/Desktop")
+Example paths on Ubuntu:
+perceval_path: /home/myComputer/.local/bin/perceval
+git_repo_path: ~/Documents/GitHub/thrift/.git
+
+Example paths on Windows: (there seems to be some issues to run perceval in Windows)
+perceval_path: C:/Users/myComputer/AppData/Local/Packages/PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0/LocalCache/local-packages/Python310/Scripts/perceval
+git_repo_path: C:/Users/myComputer/Documents/GitHub/thrift/.git
```

Comment:
I am not sure we should include this either to the notebook, it seems function-specific instead.